addons/website/static/src/client_actions/configurator/configurator.scss

```diff
@@ -1,7 +1,7 @@
 .o_configurator_container {
     @include o-position-absolute(0, 0, 0, 0);
     background-color: $o-view-background-color;
-    $input-font-size: 40%;
+    $input-font-size: 50%;
```

Comment:
Keep 40%, it was a nice compromise, other langs will be longer, don't try to fit the whole space pixel perfect in english :+1:

Reply:
Ok i'll keep it that way